sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -2356,14 +2358,9 @@ public async Task StartRecognizeCustomFormsThrowsForDamagedFile(bool useTraining
             using var stream = new MemoryStream(damagedFile);
 
             await using var trainedModel = await CreateDisposableTrainedModelAsync(useTrainingLabels);
-            var operation = await client.StartRecognizeCustomFormsAsync(trainedModel.ModelId, stream);
-
-            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () => await operation.WaitForCompletionAsync(PollingInterval));
-
-            Assert.AreEqual("2005", ex.ErrorCode);
 
-            Assert.True(operation.HasCompleted);
-            Assert.False(operation.HasValue);
+            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () => await client.StartRecognizeCustomFormsAsync(trainedModel.ModelId, stream));
+            Assert.AreEqual("1000", ex.ErrorCode);
```

Comment:
Behavior change here makes it closer to what other FR APIs do (throwing when starting the operation instead of when checking operation status). According to the service team, this change happens because now they're throwing the error during content type detection on their side. The error code still differs depending on the API, though (three different error codes: one for content recognition, one custom form recognition, and one for prebuilt model recognition).

I wonder if it's worth improving these tests a bit to check what happens when we pass a damaged file AND specify the file type.

Reply:
looks like related to https://github.com/Azure/azure-sdk-for-net/issues/19375 right?
I'm not sure it is worth it to check the other error codes. At that point we will be testing the service and not the SDK, so maybe just leave one test.

But because the exception behavior changed, I think we should mention this in our breaking change section in the changelog

Reply:
> looks like related to #19375 right?

Wow, now the issue name makes sense. I've always wondered what it was about. It's the same behavior change, but I'm not sure about the cause being the same.

You're right, this is a breaking change. I'll add it to the changelog and create an issue to follow up on that.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/19829